syntax = "proto3";
package quantum;

service QuantumService {
  rpc RunQuil(RunQuilRequest) returns (stream QuilResult);
}

message RunQuilRequest {
  string program = 1;
  uint32 shots = 2;
  map<string, double> params = 3;
}

message QuilResult {
  repeated int32 ro = 1;
}
